contracts/arbitrum/connectors/hop/main.sol

```diff
@@ -0,0 +1,82 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @title Hop.
+ * @dev Cross chain Bridge.
+ */
+
+import { TokenInterface, MemoryInterface } from "../../common/interfaces.sol";
+import { Stores } from "../../common/stores.sol";
+import "./interface.sol";
+import "./helpers.sol";
+import "./events.sol";
+
+abstract contract Resolver is Helpers {
+	/**
+	 * @dev Bridge Token.
+	 * @notice Bridge Token on HOP.
+	 * @param params BridgeParams struct for bridging
+	 * @param bridgeToL1 bool to check which layer to migrate to
+	 * @param getId ID to retrieve amount from last spell.
+	 */
+	function bridge(
+		BridgeParams memory params,
+		bool bridgeToL1,
```

Comment:
@pradyuman-verma can't we decide this if the chainId is 1?

Reply:
Ohh, right, we can do this.